awx_collection/plugins/modules/tower_project_update.py

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/python
+# coding: utf-8 -*-
+
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.0',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = '''
+---
+module: tower_project_update
+author: "Sean Sullivan (@sean-m-sullivan)"
+short_description: Sync a Project in Ansible Tower
```

Comment:
We use both the Sync and Update terms in the module and its name. We should pick one.

Reply:
Will update that.